source/dem/particle_particle_heat_transfer.cc

```diff
@@ -0,0 +1,225 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025 The Lethe Authors
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+#include <dem/particle_particle_heat_transfer.h>
+
+double
+calculate_corrected_contact_radius(const double effective_radius,
+                                   const double effective_youngs_modulus,
+                                   const double effective_real_youngs_modulus,
+                                   const double normal_force_norm)
+{
+  const double contact_radius = pow((3 * normal_force_norm * effective_radius) /
+                                      (4 * effective_youngs_modulus),
+                                    (1.0 / 3.0));
+
+  // In certain cases, the simulation effective Young’s modulus
+  // can be chosen a few orders lower than the real effective Young’s modulus
+  // to ensure simulation stability (related to Rayleigh's time).
+  // As a consequence, the contact radius is overestimated, so a correctional
+  // factor is applied here.
+  return contact_radius *
+         pow(effective_youngs_modulus / effective_real_youngs_modulus,
+             1.0 / 5.0);
+}
+
+double
+calculate_macrocontact_resistance(const double harmonic_particle_conductivity,
+                                  const double contact_radius)
+{
+  return 1 / (2.0 * contact_radius * harmonic_particle_conductivity + DBL_MIN);
+}
+
+double
+calculate_microcontact_resistance(const double equivalent_surface_slope,
+                                  const double equivalent_surface_roughness,
+                                  const double effective_microhardness,
+                                  const double contact_radius,
+                                  const double harmonic_particle_conductivity,
+                                  const double maximum_pressure)
+{
+  return 1.184 /
```

Comment:
1.184 comes from a simplification of the equation? If so: 

Could you write what 1.184 equal to in a comment? 
e.g.
```suggestion
  // 1.184 = PI / 3   
  return 1.184 
```

I know PI/3 is not equal to 1.184. but I think you get the point.

Reply:
Actually I think is a factor coming from the heat transfer model

Reply:
Then it's fine!